### ISSUE TITLE
Links forwarded to godoc.org instead of pkg.go.dev

### ISSUE THREAD

**Issue by meliache** [go-hep/go-hep.org#1]
Under 
> `Go-HEP` currently sports the following packages

the go packages link to `go-hep.org/x/hep/...`, but when I click those it seems they are forwarded to godoc.org, which is not found, so I get stuck on the package saying
> Nothing to see here; [move along](https://godoc.org/go-hep.org/x/hep/brio).

When I change the `godoc.org` to `pkg.go.dev` in the URL that [move along](https://godoc.org/go-hep.org/x/hep/brio) points to, it works. So I would change this and other occurences of godoc (maybe just `git grep godoc`).

**Comment by sbinet** [comment]
there should be a redirect to `godoc.org/...` which should itself redirect to `pkg.go.dev/...`.
doesn't this work?
(what's your browser+OS ?)

**Comment by sbinet** [comment]
(anyways: I migrated to `pkg.go.dev` through out. thanks for the report :P)

**Comment by meliache** [comment]
Arch Linux with Kernel 5.16.14, I had reproduced it with Mozilla Firefox 98.0.1 and Chromium 99.0.4844.51 Arch Linux. However, _now_ the `godoc.org` links seem to be working for me, which wasn't the case before, this is weird :thinking:. Could this be something on server-side or is it something with my computer or my connection, e.g. some DNS issues or something like this? Anyway, thanks for the fast response.

Also I must note that I did a system update & reboot today, so I'm not sure my browser versions were the same when I first tried go-hep.org and now.
